flytekit/bin/entrypoint.py

```diff
@@ -219,7 +219,7 @@ def setup_execution(
                 "api_version": _api_version,
             },
         ),
-        logging=_logging,
+        logging=python_logging,
```

Comment:
this is the handler passed to the user - will preserve the old behavior so that users don't see changes.